bin/test_travis.sh

```diff
@@ -100,13 +101,16 @@ ${CXX} -std=c++0x $compile_flags expand1.cpp $link_flags
 export LD_LIBRARY_PATH=$our_install_dir/lib:$LD_LIBRARY_PATH
 ./a.out
 
-echo "Checking whether all header files are installed:"
-python $SOURCE_DIR/symengine/utilities/tests/test_make_install.py $our_install_dir/include/symengine/ $SOURCE_DIR/symengine
-
+echo $SOURCE_DIR
+cd $SOURCE_DIR
```

Comment:
since white-space check at _last_ is not working I'll have to revert the commit which added these lines anyways.